PlanPluginBridge/src/main/java/com/djrapitops/pluginbridge/plan/nucleus/NucleusData.java

```diff
@@ -0,0 +1,311 @@
+/*
+ * Licence is provided in the jar as license.yml also here:
+ * https://github.com/Rsl1122/Plan-PlayerAnalytics/blob/master/Plan/src/main/resources/license.yml
+ */
+package com.djrapitops.pluginbridge.plan.nucleus;
+
+import com.djrapitops.plan.api.PlanAPI;
+import com.djrapitops.plan.data.element.AnalysisContainer;
+import com.djrapitops.plan.data.element.InspectContainer;
+import com.djrapitops.plan.data.element.TableContainer;
+import com.djrapitops.plan.data.plugin.ContainerSize;
+import com.djrapitops.plan.data.plugin.PluginData;
+import com.djrapitops.plan.utilities.FormatUtils;
+import com.djrapitops.plan.utilities.html.Html;
+import com.djrapitops.plan.utilities.html.HtmlUtils;
+import com.djrapitops.plan.utilities.html.icon.Color;
+import com.djrapitops.plan.utilities.html.icon.Family;
+import com.djrapitops.plan.utilities.html.icon.Icon;
+import io.github.nucleuspowered.nucleus.api.NucleusAPI;
+import io.github.nucleuspowered.nucleus.api.nucleusdata.*;
+import io.github.nucleuspowered.nucleus.api.service.*;
+import java.util.*;
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.entity.living.player.User;
+import org.spongepowered.api.service.user.UserStorageService;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.serializer.TextSerializers;
+
+/**
+ * PluginData for Nucleus plugin.
+ *
+ * @author Vankka
+ */
+public class NucleusData extends PluginData {
+    private UserStorageService userStorageService = null;
+
+    public NucleusData() {
+        super(ContainerSize.TWO_THIRDS, "Nucleus");
+        setPluginIcon(Icon.called("flask").of(Color.DEEP_ORANGE).build());
+
+        Sponge.getServiceManager().provide(UserStorageService.class).ifPresent(storageService -> userStorageService = storageService);
+    }
+
+    @Override
+    public InspectContainer getPlayerData(UUID uuid, InspectContainer inspectContainer) {
+        User user = getUser(uuid);
+
+        if (user == null) {
+            inspectContainer.addValue("Data unavailable", "Could not get user data");
+            return inspectContainer;
+        }
+
+        if (NucleusAPI.getMuteService().isPresent()) {
+            NucleusMuteService muteService = NucleusAPI.getMuteService().get();
+
+            boolean muted = muteService.isMuted(user);
+            inspectContainer.addValue(getWithIcon("Muted", Icon.called("bell-slash").of(Color.DEEP_ORANGE)), muted ? "Yes" : "No");
+
+            if (muted && muteService.getPlayerMuteInfo(user).isPresent()) {
```

Comment:
`muteService.getPlayerMuteInfo(user)` should be stored in a variable to reduce possible cache/IO reads